pkg/activator/handler/concurrency_reporter_test.go

```diff
@@ -206,7 +206,8 @@ func TestStats(t *testing.T) {
 
 	for _, tc := range tt {
 		t.Run(tc.name, func(t *testing.T) {
-			s, cr, ctx, cancel := newTestStats(t, fakeClock{}, logger)
+			s, cr, ctx, cancel := newTestStats(t, fakeClock{})
+			logging.WithLogger(ctx, logger)
```

Comment:
Why not just use the logger from `SetupFakeContext`? And delete the logger above?

Reply:
are we okay with creating multiple loggers per test?

Reply:
`t.Run` call creates a separate test

Reply:
And the same in the other places.